packages/block-editor/src/components/inserter/hooks/use-block-types-state.js

```diff
@@ -25,13 +25,18 @@ import { withRootClientIdOptionKey } from '../../../store/utils';
  * @return {Array} Returns the block types state. (block types, categories, collections, onSelect handler)
  */
 const useBlockTypesState = ( rootClientId, onInsert, isQuick ) => {
+	const options = useMemo(
+		() => ( { [ withRootClientIdOptionKey ]: ! isQuick } ),
+		[ isQuick ]
+	);
 	const [ items ] = useSelect(
 		( select ) => [
-			select( blockEditorStore ).getInserterItems( rootClientId, {
-				[ withRootClientIdOptionKey ]: ! isQuick,
-			} ),
+			select( blockEditorStore ).getInserterItems(
+				rootClientId,
+				options
+			),
```

Comment:
Shouldn't we account for this in `getInserterItems`? Also the default there is an empty object on every call, so we should change that too?

Reply:
> Shouldn't we account for this in getInserterItems?

That would be ideal, but we need to figure out how. Functions have no control over the arguments they receive.

> Also the default there is an empty object on every call, so we should change that too?

Yes. We should either pass `undefined` or an `EMTPY_OBJECT`.

Reply:
Yeah, this is a good fix for now. I wonder if we should just create another private `getInserterItems` selector and avoid the options altogether. Actually this would also allow us to remove all the reusable blocks logic. We're adding these blocks, just to filter them back out in the UI :smile: